apps/chatbot/docker/app.local.Dockerfile

```diff
@@ -3,7 +3,15 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
   apt-get install -y \
-  curl
+  curl \
+  wget \
+  zip
+
+RUN wget https://github.com/rphrp1985/selenium_support/raw/main/chrome_114_amd64.deb && \
+  apt-get install -y ./chrome_114_amd64.deb && \
+  wget https://chromedriver.storage.googleapis.com/114.0.5735.90/chromedriver_linux64.zip && \
+  unzip chromedriver_linux64.zip && \
+  mv chromedriver /usr/bin/chromedriver
```

Comment:
I suggest to keep this file without chromedriver and create another Dockerfile with chromedriver.

We'll use the new Dockerfile in `create_index` service of docker compose and in the future reindex github action.

Reply:
Lets handle this in a new and appropriate branch